<!--Launches a publisher which publishes AsrObject messages in order to to "feed" the recognizer with data, even if no cameras are used. Visualizes pubished data via RViz. This file wip. @author Fabian Hanselmann, @version See SVN.-->
<launch>
  <node name="fake_data_publisher" pkg="asr_ism" type="fake_data_publisher" output="screen">

     <!--Load some parameters recorder, trainer, recognizer and fake_data_publisher have in common.-->
    <rosparam file="$(find asr_ism)/param/sqlitedb.yaml"/>

    <!--Load some parameters trainer, recognizer and fake_data_publisher have in common.-->
    <rosparam file="$(find asr_ism)/param/visualization.yaml"/>

    <!--Load some parameters record and recognizer have in common.-->
    <rosparam file="$(find asr_ism)/param/capturing.yaml"/>
    
    <!--This option enables to publish just a specific pattern or a portion of the recorded object sets in the database to be published-->
    <!--<param name="usedPattern" value="1" />
    <param name="firstUsedSet" value="90" />
    <param name="lastUsedSet" value="101" />-->

  </node>

    <node name="asr_object_database" type="asr_object_database" pkg="asr_object_database" output="screen" />

</launch>
